source/nodejs/adaptivecards-designer-app/package.json

```diff
@@ -29,10 +29,5 @@
 		"webpack": "^5.47.0",
 		"webpack-cli": "^4.7.2",
 		"webpack-dev-server": "^3.11.2"
-	},
-	"dependencies": {
```

Comment:
adaptivecards-designer-app does have dependencies on those three packages. Why remove them?

Reply:
Reverted this